Header header
uint8 state
float32[] unixLoad
float32[] cpuLoad
uint32[] memUsed
float32 cpuTemp
float64 timestamp
